### PR TITLE
Dont update lastStep if email is not confirmed

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #567

### CODE REVIEW

File: src/services/projectVerificationFormService.ts [hunk 1]
```diff
@@ -32,6 +32,30 @@ import { errorMessages } from '../utils/errorMessages';
 import { ProjectVerificationUpdateInput } from '../resolvers/types/ProjectVerificationUpdateInput';
 import { removeUndefinedFieldsFromObject } from '../utils/utils';
 
+const updateLastStep = async (params: {
+  projectVerificationForm: ProjectVerificationForm;
+  step: string;
+}): Promise<ProjectVerificationForm> => {
+  const { step, projectVerificationForm } = params;
+  if (
+    step === PROJECT_VERIFICATION_STEPS.PERSONAL_INFO &&
```
Comment (CarlosQ96) [inline src/services/projectVerificationFormService.ts:41]: Can you skip steps or frontend validates? Maybe this validation of email_confirmed, should be present in future steps or in the submit step too.

Comment (CarlosQ96) [reply src/services/projectVerificationFormService.ts:41]: Just in case, of some wacky url manipulation.

Comment (CarlosQ96) [reply src/services/projectVerificationFormService.ts:41]: Thanks for explaning me by DM :D

Comment (mohammadranjbarz) [reply src/services/projectVerificationFormService.ts:41]: Thank you for paying attention to details and security issues

**Pull request merged**